base/exports.jl

```diff
@@ -1009,6 +1009,8 @@ export
 # dates
     Date,
     DateTime,
+    DateFormat,
+    @dateformat_str,
```

Comment:
not talking about doc/manual, talking about doc/stdlib for the docstring. yes there is a list, again see contributing.md